plugin/src/main/java/com/denizenscript/denizen/events/block/FurnaceStartsSmeltingScriptEvent.java

```diff
@@ -44,6 +45,11 @@ public class FurnaceStartsSmeltingScriptEvent extends BukkitScriptEvent implemen
 
     public FurnaceStartsSmeltingScriptEvent() {
         registerCouldMatcher("furnace starts smelting <item>");
+        this.<FurnaceStartsSmeltingScriptEvent, DurationTag>registerDetermination(null, DurationTag.class, (evt, context, time) -> {
+            if (time.asElement().isInt()) {

```

Comment:
this still has no error handling that was previously available. Go test a bad determination on an old build vs your version and compare the results